mne/viz/_3d.py

```diff
@@ -682,3 +683,79 @@ def plot_sparse_source_estimates(src, stcs, colors=None, linewidth=2,
     surface.actor.property.shading = True
 
     return surface
+
+
+def plot_dipoles(dipoles, fwd, subject, subjects_dir, bgcolor=(1, ) * 3,
```

Comment:
We should keep the function here, but have `Dipole.plot(...)` wrap to this function (it's what we do with all our plotting functions). Then you can add a simple test with `@requires_mayavi` to plot a dipole fit, probably the one we have from the testing dataset.